picky-asn1/src/wrapper.rs

```diff
@@ -379,8 +394,14 @@ impl IntegerAsn1 {
         }
     }
 
+    // TODO: This method should probably take a `&[u8]`:
```

Comment:
We can’t take owership of bytes anymore in this case though. Is `start == 0` not common enough? Also, do you have code in mind requiring keeping the `bytes` at call site? In my experience the input object is never used again once the IntegerAsn1 struct is built.